src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -9,6 +9,7 @@ const fetchStream = require('fetch-readablestream')
 const { forOwn } = require('lodash')
 const { decode: decodeBase64 } = require('@stablelib/base64')
 const JSZip = require('jszip')
+require('abortcontroller-polyfill/dist/polyfill-patch-fetch')
```

Comment:
could you move this import to the `polyfills.js` file?

Reply:
done